packages/notification-service/src/firebase.ts

```diff
@@ -260,17 +266,42 @@ export async function sendNotification(
   }
 
   try {
-    console.info(NOTIFICATIONS_TAG, 'Sending notification to:', address, title)
     const response = await admin.messaging().send(message, NOTIFICATIONS_DISABLED)
-    console.info('Successfully sent notification for :', address, response)
+    console.info(
+      JSON.stringify({
+        action: 'NOTIFICATION_SEND_SUCCESS',
+        type: data.type,
+        title,
+        address,
+        response,
+      })
+    )
+    analytics?.track({
+      anonymousId: 'notification-service',
+      event: 'push_notification_sent',
+      properties: {
+        timestamp: Date.now() / 1000,
```

Comment:
I don't think this is right. Timestamp in BigQuery refers to a different format (not UNIX timestamp), which is why this will fail with a UNIX timestamp. You can just make this column an integer if you want to store UNIX timestamp.

As an aside, I don't think you actually need to store the timestamp if you're using Segment because they do it for you.

Reply:
Yeah, not sure why I put the timestamp there, we definitely don't need another copy of it